src/styles/blocks/page.scss

```diff
@@ -0,0 +1,13 @@
+html {
+  height: 100%;
+}
+
+.page {
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
The `.page` block styles are correct, but ensure that you have a corresponding HTML element with the class `page` to apply these styles. Also, check if the `height: 100%;` is necessary for the `.page` class, as it might not be needed if the content itself doesn't occupy the full height of the viewport.